spec/lob/resources/postcard_spec.rb

```diff
@@ -37,12 +37,38 @@
         description: @sample_postcard_params[:description],
         to: new_address["id"],
         message: @sample_postcard_params[:message],
-        front: "https://lob.com/postcardfront.pdf",
+        front: "https://lob.com/postcardfront.pdf"
       )
 
       result["description"].must_equal(@sample_postcard_params[:description])
     end
 
+    it "should accept an idempotency_key" do
```

Comment:
I'm thinking this test might belong in `resource_base_spec.rb`, since it is a test of the `Resource` class rather than `Postcards`. If it were to remain on the `Postcards` class, I think it would belong on all other classes with a `create` method like `Letters`, `Checks`, etc.

But I think it makes more sense to put it in the tests for `Resource`. Any thoughts @mgartner?